distribution/build.gradle

```diff
@@ -289,7 +293,7 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
     /*****************************************************************************
      *                   Common files in all distributions                       *
      *****************************************************************************/
-    libFiles = { testDistro ->
+    libFiles = {
```

Comment:
We still reference this in `archives/build.gradle` using a method argument. Groovy (like Javascript) seems to be ok with passing arbitrary arguments to a closure. To add to that, this actually no longer needs to be a closure, since it's effectively static config now. We can just directly assign the `copySpec` to this property.